skimage/io/__init__.py

```diff
@@ -4,7 +4,9 @@
 
 """
 
-from .manage_plugins import *
+from .manage_plugins import (use_plugin, call_plugin, plugin_info,
+                             plugin_order, reset_plugins,
+                             find_available_plugins, available_plugins)
 from .sift import *
```

Comment:
why here is not the names list too?

Reply:
you could. I think i gave up on changing style, I mostly like flake8 to find python typos, and not imposing style.